src/hipscat/io/paths.py

```diff
@@ -3,41 +3,64 @@
 import os
 
 
-def pixel_directory(catalog_path="", pixel_order=0, pixel_number=0):
+def pixel_directory(
+    catalog_path="", pixel_order=0, pixel_number=None, directory_number=None
+):
     """Create path *string* for a pixel directory. This will not create the directory.
 
-    The directory name will take the form of:
+    One of pixel_number or directory_number is required. The directory name will
+    take the HiPS standard form of:
 
-        <catalog_path>/Norder<pixel_order>/Npix<pixel_number>/
+        <catalog_path>/Norder=<pixel_order>/Dir=<directory number>
+
+    Where the directory number is calculated using integer division as:
+
+        (pixel_number/10000)*10000
```

Comment:
Huh, I was thinking we would store the pixels in as few folders as possible and just keep them sequentially. I like this though. Could there be any problems with having lots of not very full folders at higher orders? Is it likely that would happen?

Reply:
This is prescribed by the HiPS standard, so I can't take any credit. We'll always be able to figure out which directory a pixel is in by performing this minimal operation, so that makes this strategy preferable to packing more densely.

At order 10, we'll only get up to Dir=12580000 (1258 subdirectories). I don't know what kind of problems it would create to have some folders with less than 10,000 sub-nodes. 

Because we're skipping the additional `Npix` directory that just has one leaf file, we're overall reducing the number of inodes pretty drastically, while avoiding some file system limitations. Looking at neowise, which is our largest available catalog:

```
/Norder/Npix/catalog.parquet:
  Norder = 5
  Npix = 39774
  parquet files = 39774
    Total inodes = 79553

/Norder/Dir/Npix.parquet:
  Norder = 5
  Dir = 32
  parquet files = 39774
    Total inodes = 39811
```

Reply:
That's a good point it's a lot less nodes than before. Yeah, I think this is the best way to do it, thanks.